src/assets/scripts/modules/download-filter/download-filter.js

```diff
@@ -18,20 +18,20 @@ function initDownloadFilter(formats) {
     ]
   };
 
-  const list = new List(component, options);
+  const list = new List($component, options);
   let sortActive
   let activeFormats = []
 
-  sort.addEventListener('click', handleSort);
-  filter.addEventListener('submit', event => event.preventDefault());
-  formatButtons.forEach(format => format.addEventListener('click', () => handleFormat(format) ))
+  $sort.addEventListener('click', handleSort);
+  $filter.addEventListener('submit', event => event.preventDefault());
+  $formatButtons.forEach(formatButton => formatButton.addEventListener('click', () => handleFormat(formatButton) ))
```

Comment:
All of these would then be `$formatButton` 😇

Reply:
updated